.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,87 @@
+name: "Bug Report [NEW]"
+description: "Report an issue with the software."
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: textarea
+    id: summary
+    attributes:
+      label: "Summary"
+      description: "Provide a short description of the issue."
+      placeholder: "e.g., App crashes when opening a file"
+    validations:
+      required: true
+
+  - type: textarea
+    id: steps_to_reproduce
+    attributes:
+      label: "Steps to Reproduce"
+      description: "Provide detailed steps to reproduce the issue."
+      placeholder: |
+        e.g., 
+        1. Open the app.
+        2. Click on the file.
+        3. The app crashes.
+    validations:
+      required: true
+
+  - type: input
+    id: os
+    attributes:
+      label: "Operating System"
+      description: "Specify the operating system you are using."
+      placeholder: "e.g., Ubuntu 24.04"
+      
+  - type: input
+    id: jellyfin_server
+    attributes:
+      label: "Jellyfin Server Version"
+      description: "Specify the version of the Jellyfin server you're using. If you're using an unstable build, please include the commit hash."
+      placeholder: "e.g., 10.9 or 10.10"
+      
+  - type: input
+    id: kodi_version
+    attributes:
+      label: "Kodi Version"
+      description: "Specify the Kodi version you're using."
+      placeholder: "e.g., 20.0"
+      
+  - type: dropdown
+    id: addon_mode
+    attributes:
+      label: "Addon Mode"
+      description: "Select how you're using the addon."
+      options:
```

Comment:
Per the docs, this *shouldn't* give a default, but it does?

Reply:
It seems to "start" on the First entry of the DropDown selected.
The only option to "fix" this, would be to have a "Please Select" option in the Dropdown.

However people could then also turn in issues with the "please select" Mode 🗿

Reply:
that's fine, I can make CI to tag the issue and yell at them if they do